application/packages/python/ftrs_data_layer/models.py

```diff
@@ -354,12 +354,21 @@ def assign_opening_times(cls, availability: dict) -> OpeningTime:
         return items if len(items) > 0 else None
 
 
+format_to_payloadMimeType = {
```

Comment:
```suggestion
payloadMimeType_mapping = {
```
To keep the enums naming conventions  in nouns or noun phrases